### PR TITLE
Feature/ks 22582

### PR DESCRIPTION
**Pull request by zellenka**
image

### CODE REVIEW

File: packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx [hunk 1]
```diff
@@ -252,13 +258,17 @@ export class KlevuProductQueryPopup {
...
+    await this.session
+      ?.query(
+        {
+          message: message,
+          klevuSettings: this.settings,
+        },
+        "send"
+      )
+      .catch((err) => {
+        this.sendMessageError = true
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx:270]: we should reset to false when the user sends message again

File: packages/klevu-ui/src/translations/en.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
   },
   "productQueryPopup": {
     "tTextFieldPlaceholder": "Enter your question here...",
+    "tAnswerError": "Uh O. There was a problem generating an answer. Please try again after some time.",
```
Comment (nikhil-narayana-klevu) [inline packages/klevu-ui/src/translations/en.json:26]: there is another file fi.json, copy the above text into google translate, translate from english to finnish and paste the translated value in the fi.json file.

File: packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx (hunk 1 above)
Comment (zellenka) [reply packages/klevu-ui/src/components/klevu-product-query-popup/klevu-product-query-popup.tsx:271]: add reset to onMessage function

File: packages/klevu-ui/src/translations/en.json (hunk 2 above)
Comment (zellenka) [reply packages/klevu-ui/src/translations/en.json:26]: add translation

**Review by nikhil-narayana-klevu — APPROVED** [review]

**Pull request closed without merging**